### ISSUE TITLE
some question about dataset distillation

### ISSUE THREAD

**Issue by knightcao** [SsnL/dataset-distillation#13]
Hello, Dr Wang.
     I have some question about dataset distillation. For image x^1 in the synthetic distilled training dataset,  the loss function L is very small, or even equal 0. For minimize the objective function, we obtain the distilled dataset only with x^1. As a result, the distilled dataset have only a image. 
    Or in other words, how do you controll the size of the distilled dataset.
     Thank you very much.

**Comment by SsnL** [comment]
Hi, you can specify the size of the distilled dataset with these options
https://github.com/SsnL/dataset-distillation/blob/ef46862bed45cb332389efbaa010cea4c84d7ccc/base_options.py#L261-L266

Do they satisfy your needs?

**Comment by knightcao** [comment]
Thank you for your helpful reply.
